public_root_of_trust.go

```diff
@@ -0,0 +1,89 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"os"
+
+	sigtuf "github.com/sigstore/sigstore/pkg/tuf"
+	tufclient "github.com/theupdateframework/go-tuf/client"
+)
+
+var globalBuffer bytes.Buffer
+
+type inMemoryDest struct{}
+
+func (d inMemoryDest) Write(p []byte) (n int, err error) {
+	return globalBuffer.Write(p)
+}
+
+func (d inMemoryDest) Delete() error {
+	panic("inMemoryDest delete function should not run")
+}
+
+func GetTargets(usage sigtuf.UsageKind, proxy Proxy) ([]sigtuf.TargetFile, error) {
+	// client initialization
+	var httpClient *http.Client
+	if proxy.URL != "" {
```

Comment:
when proxy is not configured(i.e. proxy.URL is ""), httpClient is nil.
Then tufclient.HTTPRemoteStore(sigtuf.DefaultRemoteRoot, nil, httpClient/nil) is called
How does it work?

Reply:
It will use the default http client.

![Screenshot 2023-08-04 at 12 40 49 AM](https://github.com/neuvector/sigstore-interface/assets/101584289/7961f68f-aba7-459f-8024-9aaf63ce5013)